src/styles/card.scss

```diff
@@ -0,0 +1,77 @@
+.catalog__card {
+  margin: 0 $card-horizontal-margin 50px;
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 15px 14px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding-top: 15px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 45px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    line-height: 15px;
+    margin-bottom: 5px;
+  }
+
+  &__product-code {
+    font-size: $font-small;
+    line-height: 13px;
+    margin-bottom: 15px;
+  }
+
+  &__rating {
+    @include space-between;
+    margin-bottom: 20px;
+    font-size: $font-small;
+  }
+
+  &__score {
+    align-self: center;
+  }
+
+  &__price-tag {
+    @include space-between;
+    align-items: baseline;
+    margin-bottom: 20px;
+  }
+
+  &__price-text {
+    font-size: $font-small;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: $font-big;
+  }
+
+  &__button {
```

Comment:
why `a` tag is named as button?

Reply:
I think it was previously in layout_product_cards task required to style button using an "a" tag.

Reply:
then name it as link, or buy-link

Reply:
Got it.